project/CelebornBuild.scala

```diff
@@ -347,7 +347,7 @@ object CelebornCommonSettings {
     PB.protocVersion := Dependencies.protocVersion,
     // set proto sources path
     Compile / PB.protoSources := Seq(sourceDirectory.value / "main" / "proto"),
-    Compile / PB.targets := Seq(PB.gens.java -> (Compile / sourceManaged).value)
+    Compile / PB.targets := Seq(PB.gens.java(Dependencies.protocVersion) -> (Compile / sourceManaged).value)
```

Comment:
The `protobuf-java` version is required. see https://github.com/thesamet/sbt-protoc/issues/360